clustered/integration-test/src/test/java/org/ehcache/clustered/management/CMClosedEventSentTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.management;
+
+import org.ehcache.CacheManager;
+import org.ehcache.Status;
+import org.ehcache.config.units.EntryUnit;
+import org.ehcache.config.units.MemoryUnit;
+import org.ehcache.management.registry.DefaultManagementRegistryConfiguration;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.terracotta.management.model.message.Message;
+import org.terracotta.management.model.notification.ContextualNotification;
+import org.terracotta.testing.rules.Cluster;
+
+import java.io.File;
+
+import static org.ehcache.clustered.client.config.builders.ClusteredResourcePoolBuilder.clusteredDedicated;
+import static org.ehcache.clustered.client.config.builders.ClusteringServiceConfigurationBuilder.cluster;
+import static org.ehcache.config.builders.CacheConfigurationBuilder.newCacheConfigurationBuilder;
+import static org.ehcache.config.builders.CacheManagerBuilder.newCacheManagerBuilder;
+import static org.ehcache.config.builders.ResourcePoolsBuilder.newResourcePoolsBuilder;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.terracotta.testing.rules.BasicExternalClusterBuilder.newCluster;
+
+public class CMClosedEventSentTest {
+
+  private static final String RESOURCE_CONFIG =
+    "<config xmlns:ohr='http://www.terracotta.org/config/offheap-resource'>"
+      + "<ohr:offheap-resources>"
+      + "<ohr:resource name=\"primary-server-resource\" unit=\"MB\">64</ohr:resource>"
+      + "<ohr:resource name=\"secondary-server-resource\" unit=\"MB\">64</ohr:resource>"
+      + "</ohr:offheap-resources>"
+      + "</config>\n"
+      + "<service xmlns:lease='http://www.terracotta.org/service/lease'>"
+      + "<lease:connection-leasing>"
+      + "<lease:lease-length unit='seconds'>5</lease:lease-length>"
+      + "</lease:connection-leasing>"
+      + "</service>";
+
+  @ClassRule
+  public static Cluster CLUSTER = newCluster().in(new File("build/cluster")).withServiceFragment(RESOURCE_CONFIG).build();
+
+  @Test(timeout = 60_000)
+  public void test_CACHE_MANAGER_CLOSED() throws Exception {
+    AbstractClusteringManagementTest.createNmsService(CLUSTER);
```

Comment:
It's weird to use the NmsService from another class like this without extending it. It feels like there should be an external NmsServiceFixture used by everythign

Reply:
yes agree. A previous committer has already widened these methods and used like this in other test. I have reused.